3rdparty/zlib/CMakeLists.txt

```diff
@@ -98,8 +99,4 @@ if(ENABLE_SOLUTION_FOLDERS)
   set_target_properties(${ZLIB_LIBRARY} PROPERTIES FOLDER "3rdparty")
 endif()
 
-if(NOT BUILD_SHARED_LIBS)
-  ocv_install_target(${ZLIB_LIBRARY} EXPORT OpenCVModules ARCHIVE DESTINATION ${OPENCV_3P_LIB_INSTALL_PATH} COMPONENT dev)
-endif()
```

Comment:
The patch breaks build with own zlib instance, e.g. static build, cross-compile, etc.

Reply:
Same reason as above.